Sources/Internal/FrameProvider.swift

```diff
@@ -150,6 +162,16 @@ final class FrameProvider {
     return CGRect(origin: CGPoint(x: x, y: y), size: daySize)
   }
 
+  func frameOfMonthFooter(_ month: Month, inMonthWithOrigin monthOrigin: CGPoint) -> CGRect {
+    let x = monthOrigin.x
+    let y = monthOrigin.y +
+        monthHeaderHeight +
+        content.monthDayInsets.top +
+        (monthsLayout.pinDaysOfWeekToTop ? 0 : (daySize.height + content.verticalDayMargin)) +
+        (CGFloat(numberOfRows(in: month)) * (daySize.height + content.verticalDayMargin))
+    return CGRect(origin: CGPoint(x: x, y: y), size: CGSize(width: monthWidth, height: monthFooterHeight ?? 0))
```

Comment:
```suggestion
    let x = monthOrigin.x
    let y = monthOrigin.y +
      monthHeaderHeight +
      content.monthDayInsets.top +
      (monthsLayout.pinDaysOfWeekToTop ? 0 : (daySize.height + content.verticalDayMargin)) +
      ((CGFloat(numberOfRows(in: month)) * daySize.height) + (CGFloat(numberOfRows(in: month) - 1) * content.verticalDayMargin)) +
      content.monthDayInsets.bottom
    return CGRect(x: x, y: y, width: monthWidth, height: monthFooterHeight ?? 0)
```

This will fix the jitter - this calculation wasn't properly taking into account the height of number of week rows and was also missing the `content.monthDayInsets.bottom`. Easy fix 🙂

Reply:
Ah great, I knew I missed something simple.

I wonder if there is a way to make this less intense, the calculations etc...

Reply:
Also, these complicated calculation are simplified a bit after I merge https://github.com/airbnb/HorizonCalendar/pull/79. Once you pull that into your branch, we can update this to be something like this:

```swift
    let numberOfWeekRows = self.numberOfWeekRows(in: month)

    let x = monthOrigin.x
    let y = monthOrigin.y +
      monthHeaderHeight +
      content.monthDayInsets.top +
      heightOfDaysOfTheWeekRowInMonth() +
      heightOfDayContent(forNumberOfWeekRows: numberOfWeekRows) +
      content.monthDayInsets.bottom
    return CGRect(x: x, y: y, width: monthWidth, height: monthFooterHeight ?? 0)
```

Reply:
That's what I am talking about! Very nice indeed!

Reply:
Once #79 is merged, I'll update this branch with the same fixes to accommodate the clean up.

Reply:
`master` has been merged into this branch, with the changes to accommodate the cleaner frame math